src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureChoicePage.jsx

```diff
@@ -0,0 +1,366 @@
+ */
+export const toxicExposureKeys = [
+  'conditions',
+  'gulfWar1990',
+  'gulfWar1990Details',
+  'gulfWar2001',
+  'gulfWar2001Details',
+  'herbicide',
+  'herbicideDetails',
+  'herbicideOtherLocations',
+  'additionalExposures',
+  'additionalExposuresDetails',
+  'specifyOtherExposures',
+];
+
+const ToxicExposureChoicePage = ({
+  goBack,
+  goForward,
+  data,
+  setFormData,
+  contentBeforeButtons,
+  contentAfterButtons,
+  onReviewPage,
+  updatePage,
+}) => {
+  const showDestructiveModal = useSelector(showToxicExposureDestructionModal);
+
+  const [
+    showDeleteToxicExposureModal,
+    setShowDeleteToxicExposureModal,
+  ] = useState(false);
+  const [
+    showDeletedToxicExposureConfirmation,
+    setShowDeletedToxicExposureConfirmation,
+  ] = useState(false);
+
+  const deletedToxicExposureConfirmationRef = useRef(null);
+
+  useEffect(
+    () => {
+      if (
+        showDeletedToxicExposureConfirmation &&
+        deletedToxicExposureConfirmationRef.current
+      ) {
+        deletedToxicExposureConfirmationRef.current.focus();
+      }
+    },
+    [showDeletedToxicExposureConfirmation],
+  );
+
+  const modalRef = useRef(null);
+  useEffect(
+    () => {
+      if (showDeleteToxicExposureModal && modalRef.current) {
+        // Use the ref directly instead of querying the document
+        scrollAndFocus(modalRef.current);
+      }
+    },
+    [showDeleteToxicExposureModal],
+  );
+
+  const hasToxicExposureData = () => {
+    return toxicExposureKeys.some(key => {
+      if (key === 'conditions') return false; // Skip conditions as it's the trigger
+      if (!(key in (data?.toxicExposure || {}))) return false;
+
+      const value = data.toxicExposure[key];
+      switch (typeof value) {
+        case 'boolean':
+          return value === true;
+        case 'string':
+          return value.trim() !== '';
+        case 'number':
+          return true;
+        case 'object':
+          if (value === null) return false;
+          if (Array.isArray(value)) return value.length > 0;
+          return Object.values(value).some(nestedValue => {
+            if (typeof nestedValue === 'boolean') return nestedValue === true;
+            if (typeof nestedValue === 'string')
+              return nestedValue.trim() !== '';
+            return !!nestedValue;
+          });
+        default:
+          return false;
+      }
+    });
+  };
+
+  const deleteToxicExposureData = () => {
+    const deepClone = cloneDeep(data);
+
+    if (deepClone.toxicExposure) {
+      toxicExposureKeys.forEach(key => {
+        if (key !== 'conditions') {
+          delete deepClone.toxicExposure[key];
+        }
+      });
+      // If user explicitly selected "none", keep it selected
+      // Otherwise, clear all conditions
+      if (data.toxicExposure?.conditions?.none === true) {
+        deepClone.toxicExposure.conditions = { none: true };
+      } else {
+        deepClone.toxicExposure.conditions = {};
+      }
+    }
+
```

Comment:
I assume it has been verified, after deleting data, that it is possible to re-add data.